app/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
@ashutoshbsathe @mhutti1 Shouldn't the header be above the `<?xml>` tag?

Reply:
It broke the build (probably build system did not consider it valid XML because the <?xml> tag wasn't found on first line)

Reply:
A xml file has to have <?xml> tag in the starting line of the file, otherwise it will not build as that is the format of the xml files. For reference I am attaching some other repo's manifest file as well.

![screen shot 2018-02-27 at 8 02 40 pm](https://user-images.githubusercontent.com/19654970/36734464-96b16fee-1bf9-11e8-85b8-45977a44a24a.png)

@brijeshshah13 

The PR is perfectly fine and it can be merged.

Reply:
@MayankChuriwal Thank you for the reference. I have already approved the changes. @mhutti1 will have a look and merge it if there are no other issues.